lighthouse-core/gather/gatherers/dobetterweb/response-compression.js

```diff
@@ -47,7 +47,7 @@ class ResponseCompression extends Gatherer {
 
       if (!isContentEncoded) {
         unoptimizedResponses.push({
-          record: record,
+          requestId: record.requestId,
```

Comment:
removed the full request `record` from the extended info as it wasn't used and I assumed it would have problems with serialization of the report JSON? Or was that just the full set of network records?

Regardless, it wasn't used except to get `requestId`, so seemed fine to remove :)